internal/bminventory/inventory.go

```diff
@@ -2542,21 +2526,17 @@ func (b *bareMetalInventory) DeregisterHost(ctx context.Context, params installe
 	return installer.NewDeregisterHostNoContent()
 }
 
-func (b *bareMetalInventory) GetHost(ctx context.Context, params installer.GetHostParams) middleware.Responder {
-	var host models.Host
+func (b *bareMetalInventory) GetHost(_ context.Context, params installer.GetHostParams) middleware.Responder {
 	// TODO: validate what is the error
-	if err := b.db.Where("id = ? and cluster_id = ?", params.HostID, params.ClusterID).
-		First(&host).Error; err != nil {
-		return installer.NewGetHostNotFound().WithPayload(common.GenerateError(http.StatusNotFound, err))
-	}
+	host, err := common.GetHostFromDB(b.db, params.ClusterID.String(), params.HostID.String())
 
-	if err := b.customizeHost(&host); err != nil {
```

Comment:
Where `customizeHost` disappeared?

Reply:
It's a cosmetic change.... I change the host to be a pointer instead of object and remove the '&'....

Reply:
will fix. probably a mistake